proof-market/application-developers/intro.mdx

```diff
@@ -0,0 +1,64 @@
+import {Card, CardSection} from '@site/src/components/CardSection'
+import NewCircuit from '@site/static/img/proof-market-icons/new-circuit.png'
+import Verify from '@site/static/img/proof-market-icons/verify.png'
+import Statements from '@site/static/img/proof-market-icons/statements.png'
+import Register from '@site/static/img/proof-market-icons/sign-up.png'
+import Request from '@site/static/img/proof-market-icons/request.png'
+import Payments from '@site/static/img/proof-market-icons/payments.png'
+import API from '@site/static/img/proof-market-icons/api.png'
+
+# Application developers
+
+## Introduction
+
+Application developers create dApps whose users can verify something on the blockchain by requesting proofs via Proof Market.
+
+Proof Market functions as a simple REST API. The basic flow of using it is as follows.
+
+1. [**Authorize the user inside Proof Market**](./tutorials/signing-up)
+2. [**Use circuits**](./tutorials/writing-new-circuit) for in-app use cases that require verification (and optionally [**verify their proofs**](./tutorials/verifying-a-circuit))
```

Comment:
not sure what this means: `for in-app use cases that require verification`

Reply:
this was an unnecessary complex way of saying 'use circuits inside your app', I have rewritten it